std_msgs/Header header

float64[7] dot_qr
float64[7] ddot_qr
float64[7] s

link_params link1
link_params link2
link_params link3
link_params link4
link_params link5
link_params link6
link_params link7

float64[7] tau_cmd
float64[7] tau_tilde

float64[6] error_pos_EE
float64[6] dot_error_pos_EE

float64 cond